src/vanna/chromadb/chromadb_vector.py

```diff
@@ -22,7 +22,8 @@ def __init__(self, config=None):
         self.embedding_function = config.get("embedding_function", default_ef)
         curr_client = config.get("client", "persistent")
         collection_metadata = config.get("collection_metadata", None)
-        self.n_results = config.get("n_results", 10)
+        self.n_results_sql = config.get("n_results", 10)
```

Comment:
Dont we want to have a `n_sql_results` and `n_document_results` which can take different values? That way, to add more documentation chunks, you wouldnt need to increase number of sql pairs, and vice versa.

Reply:
it seems to me the best solution

Reply:
This is what I mean: 

In the case that all three variables are passed to the config, I think it would make sense to have this behaviour, to best preserve backward compatibility:
* Both `n_results` and `n_sql_results` set only number of question-sql pairs to be fetched. They are essentially identical. I am not sure you want one method that set the same for all data sources by default. Maybe I am wrong.
* `n_document_results` sets only number of documents to be fetched.

Does this make sense to you, @zainhoda?

Reply:
To avoid misunderstanding, I meant in the comment earlier that your proposed solution is the ideal one

Reply:
Perhaps you can update the PR then to address this, as it seems like we are aligned? Seems rather straight forward and then I could test this as well :] But @zainhoda would need to do the final review as I am not authorized, of course.

Reply:
I should be done, if you can give me confirmation. I apologize but this is my first time contributing to a project